app/models/episode.rb

```diff
@@ -5,6 +5,15 @@ class Episode < ApplicationRecord
   has_many :timecodes
   has_and_belongs_to_many :authors
 
+  def update_youtube_info
+    return unless video_exists?(video)
+
+    @video = Yt::Video.new id: video
+    self.broadcast_begin = @video.actual_start_time
```

Comment:
`Self` is not needed here  + It looks like you don't update it in DB

Reply:
It doesn't work without self. 
I quite don't understand your point about DB update, this method works, DB updates, I have tested it many times.

Reply:
About self - you are right
But how this can change something in DB - don't clear for me